src/app/lib/utilities/preprocessor/rules/filterStraplines.js

```diff
@@ -0,0 +1,26 @@
+import pathOr from 'ramda/src/pathOr';
+
+const filterStraplines = data => {
+  const groups = pathOr(null, ['content', 'groups'], data);
+
+  if (!groups) {
+    return data;
+  }
+
+  const newGroups = groups.filter(group => {
+    const hasStrapline = pathOr(null, ['strapline', 'name'], group);
```

Comment:
Can this be simplfied to `groups.filter(group => pathOr(null, ['strapline', 'name'], group))`? Feel free to add a comment above it if its not visually clear what its doing :)

Reply:
Thank you! Fixed it:)